common/workunit/workunit.cpp

```diff
@@ -6894,11 +6894,17 @@ bool extractFromWorkunitDAToken(const char * distributedAccessToken, StringBuffe
 //   1 : Signature does not verify (wuid/username don't match, or signature does not verify)
 //   2 : Workunit not active
 //   Throws if unable to open workunit
-wuTokenStates verifyWorkunitDAToken(const char * distributedAccessToken)
+wuTokenStates verifyWorkunitDAToken(const char * _user, const char * _distributedAccessToken)
```

Comment:
style: I wouldn't prefix the parameter names with _.
Normally parameters prefixed with _ are used for constructors or set functions where the names of the parameters clash with the name of the member.  Occasionally they are also used for factory methods that call those constructors (but generally not).